apps/ledger-live-mobile/src/components/RootNavigator/types/SendFundsNavigator.ts

```diff
@@ -335,4 +336,26 @@ export type SendFundsNavigatorStackParamList = {
       | ScreenName.SendSelectDevice
       | ScreenName.SwapForm;
   };
+  [ScreenName.InternetComputerEditMemo]: {
+    accountId: string;
+    account: Account;
+    parentId?: string;
+    transaction: ICPTransaction;
+    currentNavigation:
+      | ScreenName.LendingWithdrawSummary
```

Comment:
Same here, you can remove all references to "lending" everywhere in your changes.

Reply:
Done!